web/src/components/pages/contribution/sentence-collector/write/sentence-input-and-rules/constants.ts

```diff
@@ -0,0 +1,16 @@
+import { SentenceDomain } from 'common'
+
+export const SENTENCE_DOMAIN_MAPPING: Record<SentenceDomain, string> = {
+  agriculture: 'Agriculture',
+  automotive: 'Automotive',
+  finance: 'Finance',
+  food_service_retail: 'Food, Service and Retail',
+  general: 'General',
+  healthcare: 'Healthcare',
+  history_law_government: 'History, Law and Government',
+  language_fundamentals: 'Language Fundamentals (e.g. Digits, Letters, Money)',
```

Comment:
In general, I think that this is ok, but I might make the explanations a separate string. E.g. there is only an explanation for one `language_fundamentals`, but I think there could potentially be explanations for various.